jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -719,3 +722,24 @@ function configure_server_state() {
 function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
+
+function configure_jbpm_cluster(){
+    if [ "${KIE_SERVER_JBPM_CLUSTER^^}" = "TRUE" ]; then
+      configure_jbpm_cache
+      log_info "KIE Server's cluster for Jbpm failover is enabled."
+    else
+      log_info "KIE Server's cluster for Jbpm failover is disabled."
+    fi
+}
+
+function configure_jbpm_cache() {
+    sed -i 's#<cache-container name="server" aliases="singleton cluster" default-cache="default" module="org.wildfly.clustering.server">#<cache-container name="jbpm">\
+        <transport lock-timeout="'${KIE_SERVER_JBPM_CLUSTER_TRANSPORT_LOCK_TIMEOUT}'"/>\
```

Comment:
I think you dismissed my review here.
what happens if the user doesn't set anything? I am not seeing a default value here.

do something like this:

```
timeout=\""${KIE_SERVER_JBPM_CLUSTER_TRANSPORT_LOCK_TIMEOUT:6000}"\"/>
```

Reply:
Added